index.html

```diff
@@ -1015,103 +1053,170 @@
 whether systems intended to protect server-side parties from abuse can collect more data
 from users.
 
+Discuss what limits need to be placed on such practices to distinguish them from privatised
+law enforcement and intellligence agencies.
```

Comment:
If you want us to address this, please add it to #73, not just inline.

Reply:
Good point — I've put this in a comment on #105.